docs/operating-scylla/nodetool-commands/refresh.rst

```diff
@@ -29,16 +29,18 @@ Load and Stream
 
 .. code::
 
-   nodetool refresh <my_keyspace> <my_table> [--load-and-stream | -las] [--scope <scope>]
+   nodetool refresh <my_keyspace> <my_table> [--load-and-stream | -las [[--primary-replica-only | -pro] | [--scope <scope>]]]
 
-The Load and Stream feature extends nodetool refresh. The new ``-las`` option loads arbitrary sstables that do not belong to a node into the cluster. It loads the sstables from the disk and calculates the data's owning nodes, and streams automatically.
+The Load and Stream feature extends nodetool refresh. The ``-las`` option loads arbitrary sstables that do not belong to a node into the cluster. It loads the sstables from the disk and calculates the data's owning nodes, and streams automatically.
 For example, say the old cluster has 6 nodes and the new cluster has 3 nodes. We can copy the sstables from the old cluster to any of the new nodes and trigger the load and stream process.
 
 Load and Stream make restores and migrations much easier:
 
 * You can place sstable from every node to every node
 * No need to run nodetool cleanup to remove unused data
 
+With --primary-replica-only (or -pro) option, only the primary replica of each partition in an sstable will be used as the target. This requires repair to be run after the load and stream operation is completed. 
```

Comment:
Need documentation on this option interacts with --scope. Although it is about to changing soon to make them composable, we should have correct documentation to backport to older versions